src/main/webapp/populateData.js

```diff
@@ -124,7 +124,7 @@ function populateCalendar() {
         const panelContent = document.querySelector('#panel-content');
         panelContent.innerHTML = '';
         for (const day in hoursJson.workTimeFree) {
-          if (typeof day == 'string') {
+          if (typeof day === 'string') {
```

Comment:
What else could `day` be? That looks a bit weird, but I'm okay with it if it's the simplest way.

Reply:
I thought there was a different check that you were doing, as in the other PR?

Reply:
The code here got reverted. I don't know what hapened exactly, but I re-implemented the check